packages/plugins/Gfi/src/store/actions/setupMultiSelection.ts

```diff
@@ -0,0 +1,94 @@
+import { Map } from 'ol'
+import { Modify } from 'ol/interaction'
+import Draw, {
+  createBox,
+  type Options as DrawOptions,
+} from 'ol/interaction/Draw'
+import { platformModifierKeyOnly } from 'ol/events/condition'
+import { Fill, Stroke, Style } from 'ol/style'
+import { PolarActionContext } from '@polar/lib-custom-types'
+import { GfiGetters, GfiState } from '../../types'
+
+const isDrawing = (map: Map) =>
+  map
+    .getInteractions()
+    .getArray()
+    .some(
+      (interaction) =>
+        (interaction instanceof Draw &&
+          // @ts-expect-error | internal hack to detect it from @polar/plugin-gfi and @polar/plugin-draw
+          (interaction._isMultiSelect || interaction._isDrawPlugin)) ||
+        interaction instanceof Modify ||
+        // @ts-expect-error | internal hack to detect it from @polar/plugin-draw
+        interaction._isDeleteSelect ||
+        // @ts-expect-error | internal hack to detect it from @polar/plugin-measure
+        interaction._isMeasureSelect
+    )
+
+const drawOptions: DrawOptions = {
+  stopClick: true,
+  type: 'Circle',
+  style: new Style({
+    stroke: new Stroke({ color: 'white', width: 1.5 }),
+    fill: new Fill({ color: [255, 255, 255, 0.75] }),
+  }),
+  freehandCondition: (event) => {
+    if (event.type === 'pointermove') {
+      return false
+    } else if (event.type === 'pointerup') {
+      return true
+    }
+    return platformModifierKeyOnly(event)
+  },
+  condition: () => false,
+}
+
+export function setupMultiSelection({
+  dispatch,
+  getters: {
+    gfiConfiguration: { boxSelect, directSelect, multiSelect },
+  },
+  rootGetters: { map },
+}: PolarActionContext<GfiState, GfiGetters>) {
+  if (boxSelect || multiSelect === 'box' || multiSelect === 'circle') {
+    if (boxSelect) {
+      console.warn(
+        '@polar/plugin-gfi: Configuration parameter "boxSelect" has been deprecated. Please use the new parameter "multiSelect" set to "box" instead.'
+      )
+    }
+    if (multiSelect !== 'circle') {
+      drawOptions.geometryFunction = createBox()
+    }
```

Comment:
```suggestion
    if (multiSelect !== 'circle') {
      drawOptions.geometryFunction = createBox()
    }
    else {
      delete drawOptions.geometryFunction;
    }
```
Just in case we ever have switchable select logic at run-time.

Reply:
Done dbb2cec92897d2881a272bebaaf6f3b3e506fce6